samples/Linq/SvgExample/StronglyTypedTools.cs

```diff
@@ -22,6 +22,11 @@ public static void AddSvg(Stream stream, string svgPath, double percentageOfCy)
             PresentationPart presentationPart = presentationDocument.PresentationPart ??
                                                 throw new InvalidOperationException(@"PresentationDocument is invalid.");
 
+            if (presentationPart.Presentation is null)
+            {
+                throw new ArgumentNullException(@"Presentation root element is missing!");
```

Comment:
Nit, but you don't need the @ symbol when there are no characters to be escaped in a string.

Reply:
I'll change both lines.  I just copied that from above and didn't remove the superfluous `@`.